kDrive/UI/Controller/Menu/Share/PublicShareViewModel.swift

```diff
@@ -28,9 +28,10 @@ final class PublicShareViewModel: InMemoryFileListViewModel {
 
     required init(driveFileManager: DriveFileManager, currentDirectory: File? = nil) {
         guard let currentDirectory else {
-            fatalError("woops")
+            fatalError("PublicShareViewModel requires a currentDirectory to work")
```

Comment:
Already mentioned in the other PR but wouldn't it be possible to remove the optional on File ?

Reply:
It was done in the `multiSelect` PR [here](https://github.com/Infomaniak/ios-kDrive/pull/1311/files#diff-b975f8c73fc66cbdc9388fae38913b7318f087cea0ccb25cffa391db4066035d)